tests/modeladmin/tests.py

```diff
@@ -1553,17 +1553,33 @@ class ProductAdmin(ModelAdmin):
             list_display_links = None
         self.assertIsValid(ProductAdmin, ValidationTestModel)
 
-    def test_list_display_same_as_list_editable(self):
+    def test_list_display_first_item_in_list_editable(self):
         """
-        The first item in list_display can be the same as the first
-        in list_editable
+        The first item in list_display can be in list_editable, as
+        long as list_display_links is defined.
         """
         class ProductAdmin(ModelAdmin):
             list_display = ['name', 'slug', 'pub_date']
-            list_editable = ['name', 'slug']
+            list_editable = ['slug', 'name']
```

Comment:
Previously we were allowing `list_display[0] == list_editable[0]`. We can relax this to `list_display[0] in list_editable`. Making this change without the fix causes the old unit test to fail.